docs/source/conf.py

```diff
@@ -61,6 +61,8 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+# The default options for autodoc directives
+autodoc_default_options = {"member-order": "bysource"}
```

Comment:
I think `autodoc_default_options` does not belong to this commit?

Reply:
Indeed. 
So I delete `auto_default_options` to restore `config.py` and recommit the task.